manifests/params.pp

```diff
@@ -1,33 +0,0 @@
-# == Class: seed_stack::params
-#
-class seed_stack::params {
-
-  $docker_ensure            = '1.10.2*'
-
-  $zookeeper_ensure         = 'installed'
-  $zookeeper_client_addr    = '0.0.0.0'
-
-  $mesos_ensure             = '0.27.1*'
-  $mesos_listen_addr        = '0.0.0.0'
-  $mesos_cluster            = 'seed-stack'
-  $mesos_resources          = {}
-
-  $marathon_ensure          = '0.15.3*'
-  $marathon_default_options = { # TODO
-    'event_subscriber' => 'http_callback' # HTTP callbacks for Consular
-  }
```

Comment:
Nope, not used. At one point I wanted all the default things/parameters to be in here but that didn't really ever happen.